lib/reporters/html/index.js

```diff
@@ -1,22 +1,89 @@
+
+                if (!_.has(traversedRequests, executionId)) {
+                    // mark the current request instance as traversed
+                    _.set(traversedRequests, executionId, 1);
+
+                    // set the base assertion and cumulative test details for the current request instance
+                    _.set(result, executionId, {});
+                    _.set(netTestCounts, executionId, { passed: 0, failed: 0 });
+
+                    // set base values for overall response size and time values
+                    _.set(executionMeans, executionId, { time: { sum: 0, count: 0 }, size: { sum: 0, count: 0 } });
+
+                    // set sample request and response details for the current request
+                    aggregations.push(_.pick(currentExecution, ['item', 'request', 'response']));
+                }
+
+                executionMeans[executionId].time.sum += _.get(currentExecution, 'response.responseTime');
+                executionMeans[executionId].size.sum += _.get(currentExecution, 'response.responseSize');
+
+                ++executionMeans[executionId].time.count;
+                ++executionMeans[executionId].size.count;
+
+                _.forEach(currentExecution.assertions, function (assertion) {
+                    var aggregationResult,
+                        assertionName = assertion.assertion,
+                        isError = _.get(assertion, 'error') !== undefined,
+                        updateKey = _.get(ASSERTION_STATE, isError);
+
+                    result[executionId][assertionName] = result[executionId][assertionName] || {
+                        name: assertionName,
+                        passed: 0,
+                        failed: 0
+                    };
+                    aggregationResult = result[executionId][assertionName];
+
+                    ++aggregationResult[updateKey];
+                    ++netTestCounts[executionId][updateKey];
+                });
+            }, {});
+
+        _.forEach(aggregations, function (currentAggregation) {
+            var currentId = currentAggregation.item.id,
+                aggregationMean = _.get(executionMeans, currentId),
+                meanTime = _.get(aggregationMean, 'time'),
+                meanSize = _.get(aggregationMean, 'size');
+
+            _.merge(currentAggregation, {
+                mean: {
+                    time: (meanTime.sum / (meanTime.count || 1)).toFixed(2),
+                    size: (meanSize.sum / (meanSize.count || 1)).toFixed(2)
+                },
+                cumulativeTests: _.get(netTestCounts, currentId)
+            });
+
+            _.set(currentAggregation, 'assertions', _.values(_.get(assertions, currentId)));
+        });
+
+        _.set(this.summary.run, 'aggregations', aggregations);
+
         this.exports.push({
             name: 'html-reporter',
             default: 'newman-run-report.html',
-            path: options.export,
+            path: options.output,
```

Comment:
`options.export` is no longer being used as a valid key, we had switched to output.